src/node/elder_duties/key_section/transfers/genesis.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use crate::{ElderState, Error, Result};
+use sn_data_types::{Credit, CreditAgreementProof, Money, SignedCredit};
+use std::collections::BTreeMap;
+
+/// Produces a genesis balance for a new network.
+pub async fn get_genesis(balance: u64, elder_state: &ElderState) -> Result<CreditAgreementProof> {
+    let credit = Credit {
+        id: Default::default(),
+        amount: Money::from_nano(balance),
+        recipient: elder_state.section_public_key(),
+        msg: "genesis".to_string(),
+    };
+
+    // actor instances' signatures over > credit <
+
+    let mut credit_sig_shares = BTreeMap::new();
+    let credit_sig_share = elder_state.sign_as_elder(&credit).await?;
+    let _ = credit_sig_shares.insert(credit_sig_share.index, credit_sig_share.share);
+
+    println!("Aggregating actor signature..");
```

Comment:
do we want to keep these? perhaps switch to info?
```suggestion
    info!("Aggregating actor signature..");
```